packages/genesys-spark-components/src/components/stable/gux-accordion/gux-accordion-section/gux-accordion-section.scss

```diff
@@ -51,53 +53,37 @@ section.gux-disabled {
   ::slotted([slot='header']) {
     padding: 0;
     margin: 0;
-    font-family: var(
-      --gse-ui-accordion-header-label-defaultText-fontFamily
-    ) !important;
-    font-size: var(
-      --gse-ui-accordion-header-label-defaultText-fontSize
-    ) !important;
-    font-weight: var(
-      --gse-ui-accordion-header-label-defaultText-fontWeight
-    ) !important;
-    line-height: var(
-      --gse-ui-accordion-header-label-defaultText-lineHeight
-    ) !important;
-    color: var(--gse-ui-accordion-header-default-foreground-labelColor);
+    font-family: ui.$gse-ui-accordion-header-label-defaultText-fontFamily !important;
+    font-size: ui.$gse-ui-accordion-header-label-defaultText-fontSize !important;
+    font-weight: ui.$gse-ui-accordion-header-label-defaultText-fontWeight !important;
+    line-height: ui.$gse-ui-accordion-header-label-defaultText-lineHeight !important;
+    color: ui.$gse-ui-accordion-header-default-foreground-labelColor;
   }
 
   ::slotted([slot='subheader']) {
     // TODO COMUI-2370. Missing tokens for subheaders text styling.
```

Comment:
Is this PR covering this comment so it can be removed and tied ticket closed? I see there are text styling tokens being used now

Reply:
No but it looks like it was missed when it was fixed so I will remove now